xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -371,6 +378,13 @@ rendering.transformations = $xwikiRenderingTransformations
 #-# If no path is provided, a default value will be calculated based on the operating environment.
 # openoffice.profilePath = /home/user/.openoffice.org/3
 
+#-# [Since 16.10.0RC1]
+#-# [Since 16.4.6]
+#-# Path to a folder where XWiki will exchange temporary files with the openoffice server.
+#-# If no path is provided, the temporary folder set for XWiki will be used.
+#-# If openoffice.serverType = 2, this folder should be a shared volume between the two servers.
```

Comment:
should or must?

Reply:
> If openoffice.serverType = 2, this folder should be a shared volume between the two servers.

Hmmm I missed that @gdelhumeau if your plan is to have a shared volume, then why shouldn't you consider the LO as externally but locally installed (serverType=1)?